lms/djangoapps/bulk_email/tasks.py

```diff
@@ -95,29 +95,35 @@
 )
 
 
-def _get_course_email_context(course):
+def _get_course_email_context(course, site_context):
```

Comment:
I would use something like:
```suggestion
def _get_course_email_context(course, site_context={}):
```
Since we are not going to change `site_context` it is safe to do it. That way we can have a default and avoid some `if` later

See https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python